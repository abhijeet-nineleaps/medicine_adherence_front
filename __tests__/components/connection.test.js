

import React from "react";
import enableHooks from "jest-react-hooks-shallow";
import Enzyme, { shallow,render } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
Enzyme.configure({ adapter: new Adapter() });
import checkConnectivity from '../../src/connection/checkConnectivity'
import toJson from "enzyme-to-json";
enableHooks(jest);

jest.mock("@react-navigation/native", () => ({
  ...jest.requireActual("@react-navigation/native"),
  useFocusEffect: jest.fn(),
  useEffect: jest.fn(),
}));
jest.mock('@react-native-google-signin/google-signin/lib/commonjs', () => ({
  default: jest.fn(),
}));
jest.mock('@react-native-community/netinfo', () => ({
    default: jest.fn(),
  }));
describe('check connectivity', () => {
  it("test", () => {
    const wrapper = shallow(<checkConnectivity />);
    expect(toJson(wrapper)).toMatchSnapshot();
  })
});