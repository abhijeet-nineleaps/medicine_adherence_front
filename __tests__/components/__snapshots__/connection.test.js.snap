// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check connectivity test 1`] = `<checkConnectivity />`;
